lib/src/ui/movie_detail.dart

```diff
@@ -0,0 +1,156 @@
+                            fontSize: MovieDimensions.subtitleFontSize,
+                            fontWeight: FontWeight.bold,
+                          ),
+                        ),
+                        SizedBox(
+                          height: MovieDimensions.sizedBoxHeight,
+                        ),
+                        Row(
+                          children: [
+                            Text(
+                              result.voteAverage.toString(),
+                              style: TextStyle(
+                                color: Colors.yellow,
+                                fontSize: MovieDimensions.subtitleFontSize,
+                              ),
+                            ),
+                            SizedBox(
+                              width: MovieDimensions.sizedBoxWidth,
+                            ),
+                            ...List.generate(
+                              MovieDimensions.starsAmount,
+                              (
+                                index,
+                              ) =>
+                                  Icon(
+                                Icons.star,
+                                color:
+                                    (index < (result.voteAverage / 2).floor())
+                                        ? Colors.yellow
+                                        : Colors.white,
+                              ),
+                            ),
+                          ],
+                        ),
+                      ],
+                    ),
+                  ),
+                ),
+              ),
+              SizedBox(
+                height: MovieDimensions.sizedBoxHeight,
+              ),
+              Padding(
+                padding: const EdgeInsets.only(
+                  left: MovieDimensions.edgeInsetsLeftPadding,
+                ),
+                child: Text(
+                  MovieStrings.overview,
+                  style: TextStyle(
+                    color: MovieColors.customWhite.withOpacity(
+                      MovieDimensions.overviewTextOpacity,
+                    ),
+                    fontSize: MovieDimensions.overviewTextFontSize,
+                    fontWeight: FontWeight.bold,
+                  ),
+                ),
+              ),
+              Padding(
+                padding: const EdgeInsets.only(
+                    left: MovieDimensions.edgeInsetsLeftPadding,
+                    top: MovieDimensions.edgeInsetsTopPadding),
+                child: Text(
+                  result.overview,
+                  style: TextStyle(
+                    color: MovieColors.customWhite
+                        .withOpacity(MovieDimensions.textStyleOpacity),
+                    fontSize: MovieDimensions.subtitleFontSize,
+                    fontWeight: FontWeight.bold,
+                  ),
+                ),
+              ),
+            ],
+          ),
+        ),
+      ),
```

Comment:
This can be added into a new Widget for overview

Reply:
Done, thank you